media/live-seekable-range.html

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
It works as expected as I've added `check_min_version = true` at https://github.com/GoogleChrome/samples/pull/536/files#diff-14e684df0f667174c42ca3d4b20f7a15R5
It is just a way for user to know that this sample has been tested with Chrome 62 (which I did)